README.rst

```diff
@@ -28,26 +39,24 @@ With this tracker you can collect event data from your Python-based applications
-| Technical Docs                  | Setup Guide               | Roadmap                 | Contributing                      |
-+=================================+===========================+=========================+===================================+
-| |techdocs|_                     | |setup|_                  | |roadmap|               | |contributing|                    |
-+---------------------------------+---------------------------+-------------------------+-----------------------------------+
-| `Technical Docs`_               | `Setup Guide`_            | `Roadmap`_              | `Contributing`_                   |
-+---------------------------------+---------------------------+-------------------------+-----------------------------------+
++---------------------------------+---------------------------+-----------------------------------+
+| Technical Docs                  | Setup Guide               | Contributing                      |
++=================================+===========================+===================================+
+| |techdocs|_                     | |setup|_                  | |contributing|                    |
++---------------------------------+---------------------------+-----------------------------------+
+| `Technical Docs`_               | `Setup Guide`_            | `Contributing`_                   |
++---------------------------------+---------------------------+-----------------------------------+
 
 .. |techdocs| image:: https://d3i6fms1cm1j0i.cloudfront.net/github/images/techdocs.png
 .. |setup| image:: https://d3i6fms1cm1j0i.cloudfront.net/github/images/setup.png
-.. |roadmap| image:: https://d3i6fms1cm1j0i.cloudfront.net/github/images/roadmap.png
 .. |contributing| image:: https://d3i6fms1cm1j0i.cloudfront.net/github/images/contributing.png
 
-.. _techdocs: https://github.com/snowplow/snowplow/wiki/Python-Tracker
-.. _setup: https://github.com/snowplow/snowplow/wiki/Python-Tracker-Setup
+.. _techdocs: https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/python-tracker/
+.. _setup: https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/python-tracker/setup/
 
-.. _`Technical Docs`: https://github.com/snowplow/snowplow/wiki/Python-Tracker
-.. _`Setup Guide`: https://github.com/snowplow/snowplow/wiki/Python-Tracker-Setup
-.. _`Roadmap`: https://github.com/snowplow/snowplow/wiki/Python-Tracker-Roadmap
-.. _`Contributing`: https://github.com/snowplow/snowplow/wiki/Python-Tracker-Contributing
+.. _`Technical Docs`: https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/python-tracker/
+.. _`Setup Guide`: https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/python-tracker/setup/
+.. _`Contributing`: https://github.com/snowplow/snowplow-python-tracker/blob/master/CONTRIBUTING.md
 
 Quickstart
```

Comment:
I've recently started updating this title to `Maintainer Quickstart` to remove any confusion that you need to do this to use the tracker.